app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ReviewsController < ApplicationController
+  respond_to :js
+  def create
+    @review = Review.new(review_params)
```

Comment:
you can just you load resource from cancancan

Reply:
done